app/views/parks/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1>Hello from parks show!</h1>
 
-<div id="app" data-id= <%= @id %> >
+<div id="app" data-park_id= <%= @id %> data-user_id= <%= current_user && current_user.id %> >
```

Comment:
The inconsistence here, in your data attribute, between hyphens and underscores, is a *little* maddening. Why not `data-user-id` (and then reference it as such)?